metadata.json

```diff
@@ -27,7 +27,8 @@
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
         "18.04",
-        "20.04"
+        "20.04",
+        "21.04"
```

Comment:
do you have a ubuntu 21.04 box around to provide factsets for https://github.com/voxpupuli/facterdb ? Also please create a new PR that adds 21.04 here. That allows us to put the two PRs in different sections (breaking change vs enhancement)